docs/content/docs/data-structures/sequence.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Sequences and Strings
+menuPosition: 3
+---
+
+<!-- IMPORTANT: Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path
+     argument) and then run "npm run readme:update" from the root of the repo. That will update all autogenerated
+     content, so you should commit the resulting changes. -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_ROOT:path=packages/dds/sequence/README.md&start=2) -->
+Every item in a SharedSegmentSequence is at a specific position starting at 0, kind of like an array. However, it
+differs from an array in that the positions can move as local and remote collaborators make modifications to the
+sequence. There are a number of different sequence types:
+
+- SharedString for storing and collaborating on a sequence of text.
+- SharedNumberSequence for storing and collaborating on a sequence of numbers.
+- SharedObjectSequence for storing and collaborating on a sequence of JSON-serializable objects.
+
+As the name suggests SharedSegmentSequence, or sequence for short, are made of segments. Segments are the leaf nodes of
+the tree data structure that enables collaboration and backs the sequence. Segments may be split and merged as
+modifications are made to the sequence. Every segment has a length from 1, to the length of the sequence. The length of
+the sequence will be the combined length of all the segments.
+
+When talking about positions in a sequence we use the terms *near*, and *far*. The nearest position in a sequence is 0,
+and the farthest position is its length. When comparing two positions the nearer position is closer to 0, and the
+farther position is closer to the length.
+
+## Using a Sequence
+
+Sequences support three basic operations: insert, remove, and annotate.
+
+Insert operations on the sequence take a single position argument along with the content. This position is inclusive.
+This position can any position in the sequence including 0, and the length of the sequence.
+
+```typescript
+    //   content:
+    // positions:
+
+    // insert text a positions 0
+    sharedString.insertText(0, "hi");
+    //   content: hi
+    // positions: 01
+
+    // insert text at the end position
+    sharedString.insertText(
+        sharedString.getLength(),
+        "!");
+    //   content: hi!
+    // positions: 012
+
+    // insert text at position 2
+    sharedString.insertText(
+        2,
+        " world");
+    //   content: hi world!
+    // positions: 012345678
+```
+
+Remove operations take a start and an end position. The start position is similar to the insert's position, in that is
```

Comment:
I feel like there is disconnect in the flow RE segments and position. I.e. till now we said that data is represented as segments and segment length is 1. We did not introduce how test is mapped to segments, so uneducated user may assume that position in this context maps to directly to segments, i.e. end-start+1 is number of segments impacted by this operation, which is not the case.

Reply:
Also probably worth clarifying that sharedString.getLength() is not number of segments, it's number of characters.
Somewhere we need to say that if you enumerate segments, some of them will be "deleted" segments.